app/views/items/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>All wooden goods</h1>
+<% @items.each do |item| %>
+  <div class="col-sm-6 col-md-4">
+    <div id="item_<%= item.id %>" class="thumbnail">
+      <%= link_to item_path(item) do %>
+        <img src=<%= item.image_url %> alt=<%= item.title %>>
+        <div class="caption">
+          <h3><%= item.title %></h3>
+        </div>
+      <% end %>
+      <button class="btn btn-info add_to_cart">Add to cart</button>
+    </div>
```

Comment:
@Robbie-Smith aforementioned .add_to_cart class. This button doesn't actually do anything at the moment.